.swiftpm/xcode/package.xcworkspace/contents.xcworkspacedata

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
you need to add this file to .gitignore file i guess. other than that looks good.

i'll assign you some ios tickets 😄